packages/opam-depext/opam-depext.1.1.2/opam

```diff
@@ -12,16 +12,18 @@ bug-reports: "https://github.com/ocaml/opam-depext/issues"
 license: "LGPL-2.1 with OCaml linking exception"
 dev-repo: "git+https://github.com/ocaml/opam-depext.git#2.0"
 build: [make]
-available: [opam-version >= "2.0.0~beta5"]
+available: opam-version >= "2.0.0~beta5"
 synopsis: "Query and install external dependencies of OPAM packages"
 description: """
 opam-depext is a simple program intended to facilitate the interaction between
 OPAM packages and the host package management system. It can query OPAM for the
 right external dependencies on a set of packages, depending on the host OS, and
 call the OS's package manager in the appropriate way to install them."""
-depends: ["ocaml"]
+depends: [
+  "ocaml" {>= "4.0.0"}
+]
 flags: plugin
 url {
-  src: "https://github.com/ocaml/opam-depext/releases/download/v1.1.2/opam-depext-full-1.1.2.tbz"
-  checksum: "md5=d71c9c0ada811ccf0669d09e1b0329da"
+  src: "https://github.com/ocaml/opam-depext/archive/v1.1.2.tar.gz"
```

Comment:
hum that seems wrong

Reply:
I looked into it and it comes from this fix on 2.0.0 only which should have probably been done on master instead: https://github.com/ocaml/opam-repository/commit/083c4b8e25211528083e2162320175fee67880f6

I'll push the fix to master and merge this one afterwards